conans/test/integration/environment/test_env.py

```diff
@@ -171,12 +173,11 @@ def test_profile_buildenv():
     conanfile = textwrap.dedent("""\
         import os, platform
         from conans import ConanFile
+        from conan.tools.env import save_script
         class Pkg(ConanFile):
             def generate(self):
-                if platform.system() == "Windows":
-                    self.buildenv.save_bat("pkgenv.bat")
-                else:
-                    self.buildenv.save_sh("pkgenv.sh")
+                save_script(self, self.buildenv, "pkgenv")
```

Comment:
as the ``Environment`` is always an arg, maybe it is a bit more idiomatic (and require 1 import less):

```
self.buildenv.save_script(self, "pkgenv")
```

Reply:
The conanfile is not needed either, because it is always there.